### DISCUSSION TITLE
"frontmax" parameter for power bars.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#2137] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

I'm trying to replicate KoF's stock system where the stock limit increases by one for every KOed team member using ModifyPlayer, and for it to be accurate I need the maxed out animation to be able to run on levell 3, level 4, and level 5.

### Describe the solution you'd like

The "frontmax" parameter would run the designated animation/sprite whenever the power is maxed out regardless of the value.

### Additional context

I tried replicating this effect using explods to draw the animation but it doesn't seem to behave correctly with ModifyPlayer. That's why I'm making this request.

**Comment by rakieldev** [comment, marked as answer]
Duplicated request 
Also added by #3215
